File: app/src/main/java/com/acelost/universalcontent/fragmentbased/content/impl/ProfileActivity.java
package com.acelost.universalcontent.fragmentbased.content.impl;

import android.support.v4.app.Fragment;

import com.acelost.universalcontent.fragmentbased.container.ActivityContainer;
import com.acelost.universalcontent.fragmentbased.content.ProfileContentFragment;

public class ProfileActivity extends ActivityContainer {

    @Override
    protected Fragment createContentFragment() {
        return new ProfileContentFragment();
    }

}

File: app/src/main/java/com/acelost/universalcontent/contentbased/content/impl/ProfileActivity.kt
package com.acelost.universalcontent.contentbased.content.impl

import com.acelost.universalcontent.contentbased.container.ActivityContainer
import com.acelost.universalcontent.contentbased.core.Content
import com.acelost.universalcontent.contentbased.core.ContentContainer

class ProfileActivity : ActivityContainer() {

    override fun createContent(): Content {
        return ProfileContent()
    }

}